### PR TITLE
chore: use webpack-chain v6.5.1

### PR DESCRIPTION
**Pull request by cexbrayat**
Vue CLI v5.0.0-alpha.0 uses `rule.resolve` in the CLI service (see https://github.com/vuejs/vue-cli/blob/v5.0.0-alpha.0/packages/@vue/cli-service/lib/config/base.js#L17-L20) which was introduced in webpack-chain v6.5.0.

See https://github.com/neutrinojs/webpack-chain/blob/master/CHANGELOG.md#v650

As the CLI service defines a dependency to webpack-chain v6.4.0, this causes issues in existing projects when trying to upgrade to the CLI v5.

Upgrading an existing project leads to `yarn build` throwing:

```
TypeError: Cannot read property 'set' of undefined
    at /Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/config/base.js:20:19
    at /Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/Service.js:240:40
    at Array.forEach (<anonymous>)
    at Service.resolveChainableWebpackConfig (/Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/Service.js:240:26)
    at PluginAPI.resolveChainableWebpackConfig (/Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/PluginAPI.js:145:25)
    at module.exports (/Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/commands/build/resolveAppConfig.js:9:22)
    at build (/Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/commands/build/index.js:147:50)
    at /Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/commands/build/index.js:89:13
    at Service.run (/Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/lib/Service.js:234:12)
    at Object.<anonymous> (/Users/ced-pro/Code/ninjasquad/vue-ebook/book-tests/node_modules/@vue/cli-service/bin/vue-cli-service.js:36:9)
```

This fixes the issue by specifying the necesseray minimal version for webpack-chain.

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Underlying tools
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**

### CODE REVIEW

**Review by fangbinwei — APPROVED** [review]

File: packages/@vue/cli-service/package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "vue-style-loader": "^4.1.2",
     "webpack": "^5.4.0",
     "webpack-bundle-analyzer": "^4.1.0",
-    "webpack-chain": "^6.4.0",
+    "webpack-chain": "^6.5.0",
```
Comment (fangbinwei) [inline packages/@vue/cli-service/package.json:79]:
```suggestion
    "webpack-chain": "^6.5.1",
```

File: yarn.lock [hunk 2]
```diff
@@ -21888,7 +21888,7 @@ webpack-chain@^4.9.0:
     deepmerge "^1.5.2"
     javascript-stringify "^1.6.0"
 
-webpack-chain@^6.0.0, webpack-chain@^6.4.0:
+webpack-chain@^6.0.0, webpack-chain@^6.5.0:
```
Comment (fangbinwei) [inline yarn.lock:21891]:
```suggestion
webpack-chain@^6.0.0, webpack-chain@^6.5.1:
```

**Review by fangbinwei — COMMENTED** [review]
bump version to 6.5.1 directly ?

File: packages/@vue/cli-service/package.json (hunk 1 above)
Comment (cexbrayat) [reply packages/@vue/cli-service/package.json]: Fixed 👍

**Pull request merged**